helm/vertical-pod-autoscaler-app/templates/crd-patch/crd-psp.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: policy/v1beta1
```

Comment:
Not needed?

Reply:
Same question as above

Reply:
If we removed the job we can remove the folder yes

Reply:
great !